### PR TITLE
Add confirmation to warn user of the cost of second passphrase

### PR DESCRIPTION
**Pull request by ckhatri**
Some people on the subreddit were mentioning there wasn't any popup or information about how creating a second passphrase costs ark. This toast warning message should popup and warn them how much it'll cost!

### CODE REVIEW

File: client/app/config/config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
   notice: {
     logFile: false,
-    level: 1,
+    level: 2,
```
Comment (j-a-m-l) [inline client/app/config/config.js]: Why this change?

Comment (ckhatri) [reply client/app/config/config.js]: The main reason for that change is because every time we send a toast message some message types are shown to the user such as error, success, but others are supposed to be logged only such as log and debug. I believe warning should be shown to the user, not logged?

Currently it does

if (type > self.loggingType) {
        self.logToFile(message, typeName)
        return
      }

Comment (j-a-m-l) [reply client/app/config/config.js]: Yes, a confirmation would be a new intermediate screen after clicking "NEXT" on the same window, or a pop-up with "I agree" or something similar.

Comment (ckhatri) [reply client/app/config/config.js]: Added an alert that shows before. They must click OK before the secondPassphrase starts.

The toast message is also there still.

Comment (j-a-m-l) [reply client/app/config/config.js]: Good work.

I'd remove the toast message because toast notices should be a way to reflect the end of an action on the background, an error, etc. In this case, is unnecessary and it could even be distracting.

Apart from that, the only change I'd do is using different texts:
"Are you sure?" => "Second passphrase fee"
"WARNING! Second passphrase creation costs ' + secondPhraseArkVal + '  '+ currency +'.'"
"CREATE" => "CONTINUE"

Comment (j-a-m-l) [reply client/app/config/config.js]: BTW @ckhatri What do you think about https://github.com/ArkEcosystem/ark-desktop/issues/431 ?

Comment (ckhatri) [reply client/app/config/config.js]: Should I keep the changes to the levels though? Since we do want toast warnings to be shown not logged? Or should I revert that back?

Agreed with the changes, I'll get those done soon.

Comment (j-a-m-l) [reply client/app/config/config.js]: I'd remove entirely the toast message and I'd revert the notice configuration to level 1.

Comment (ckhatri) [reply client/app/config/config.js]: I got rid of the toast message and reverted the notice!

**Comment by j-a-m-l** [comment]
+5 Good work!
